submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestReleaser.kt

```diff
@@ -45,12 +45,11 @@ class SubmissionRequestReleaser(
         val sub = request.submission
         filesRequestService
             .getSubmissionRequestFiles(sub.accNo, sub.version, request.currentIndex)
-            .filterNot { it.file is FireFile && (it.file as FireFile).published }
-            .map { it.copy(file = fileStorageService.releaseSubmissionFile(it.file, sub.relPath, sub.storageMode)) }
-            .forEach {
-                filesRequestService.saveSubmissionRequestFile(it)
-                requestService.updateRequestIndex(sub.accNo, sub.version, it.index)
+            .map {
```

Comment:
I just noticed we're missing the individual file logs here as in the other steps:

```
logger.info { "${sub.accNo} ${sub.owner} Started publishing file ${it.index}, path='${it.path}'" }

releaseFile

logger.info { "${sub.accNo} ${sub.owner} Finished releasing file ${it.index}, path='${it.path}'" }

```